EatGoodNaija.Server/Controllers/AuthController.cs

```diff
@@ -108,5 +110,25 @@ public async Task<IActionResult> ConfirmEmail(ConfirmEmailTokenDTO token, string
                 return BadRequest(result);
             }
         }
+
+        [HttpPut("update-profile")]
+        public async Task<IActionResult> UpdateUserProfile(UpdateProfileDTO updateProfileDTO, string Id)
+        {
+            if (!ModelState.IsValid)
```

Comment:
Your controller should have minimum line of code possible. Do all checks in your services and or repository. E.g

public async Task<IActionResult> UpdateUserProfile(UpdateProfileDTO updateProfileDTO, string Id)
        {
            if (ModelState.IsValid)
           {
                var response= await _authService.UpdateProfileAsync(updateProfileDTO, Id);
                return Ok(response);
           }